tests/expectations/swift_enum.both.c

```diff
@@ -0,0 +1,184 @@
+#import <CoreFoundation/CoreFoundation.h>
```

Comment:
These headers are not available in CI so you need to use a local macro for the test to compile or what not. Let me know if you need help with that.

Reply:
Roughly something like this should do it I think. The macro is called after the typedef keyword.

```
#define CF_ENUM(_type, _name) _type _name; enum _name
```